### ISSUE TITLE
WCS 2.0 - GeoTIFF - tiling - non-compliance.

### ISSUE THREAD

**Issue by pacesm** [EOxServer/eoxserver#357]
When the `geotiff:tiling` is set to `true` but no tile-size (neither `geotiff:tilewidth` nor `geotiff:tilewidth`) is provided the server responds with following OWS exception: 

```
InvalidParameterValue: geotiff:tiling requires geotiff:tilewidth and geotiff:tileheight to be set.
```

This behaviour is not compliant with the applicable standard. See Tab. 2, [OGC 12-100r1](https://portal.opengeospatial.org/files/?artifact_id=54813).

**Comment by constantinius** [comment]
TODOs:
- [ ] Check the standard what the behavior shall be (defaults,...)
- [ ] Check if this is even an issue anymore
- [ ] Find solution
- [ ] Add a testcase

**Comment by baloola** [comment]
fixed in #419
